### PR TITLE
ENTESB-7783 Change references to "JBoss Fuse" to "Red Hat Fuse"

### PR DESCRIPTION
**Pull request by cunningt**
https://issues.jboss.org/browse/ENTESB-7783

I've gone ahead and changed all the references to "JBoss Fuse" to "Red Hat Fuse" and changed the kit names.    When I built locally, I was able to run the kit and install quickstart examples, so I think I've done this correctly but would appreciate a second pair of eyes.

Note : changing the kit artifact name from jboss-fuse-karaf to fuse-karaf may require changes in the smoke tests / QE tests etc

### CODE REVIEW

**Comment by grgrzybek** [comment]
@oscerd , @cunningt , @janstey please don't merge yet - I have to test patching first! It has some references to top-level dirs.

File: modules/fuse-branding/src/main/resources/org/apache/karaf/branding/branding.properties [hunk 1]
```diff
@@ -15,19 +15,19 @@
...
 
 welcome = \r\n\
-\u001B[36m      _ ____                  ______              \u001B[0m\r\n\
-\u001B[36m     | |  _ \\                |  ____|             \u001B[0m\r\n\
-\u001B[36m     | | |_) | ___  ___ ___  | |__ _   _ ___  ___ \u001B[0m\r\n\
-\u001B[36m _   | |  _ < / _ \\/ __/ __| |  __| | | / __|/ _ \\ \u001B[0m\r\n\
-\u001B[36m| |__| | |_) | (_) \\__ \\__ \\ | |  | |_| \\__ \\  __/\u001B[0m\r\n\
-\u001B[36m \\____/|____/ \\___/|___/___/ |_|   \\__,_|___/\\___|\u001B[0m\r\n\
+\u001B[36m ______              \u001B[0m\r\n\
+\u001B[36m |  ____|             \u001B[0m\r\n\
```
Comment (apupier) [inline modules/fuse-branding/src/main/resources/org/apache/karaf/branding/branding.properties]: why changing to FUSE and not Red Hat Fuse?

**Comment by grgrzybek** [comment]
I'm assigning to myself - to check few internal tricks used by patching mechanism.

**Comment by grgrzybek** [comment]
Closing - changed in #81

**Comment by grgrzybek** [comment]
```
12:19 $ git push origin HEAD:refs/pull/80/head
Total 0 (delta 0), reused 0 (delta 0)
To github.com:jboss-fuse/fuse-karaf.git
 ! [remote rejected] HEAD -> refs/pull/80/head (deny updating a hidden ref)
error: failed to push some refs to 'git@github.com:jboss-fuse/fuse-karaf.git'
```

**Pull request closed without merging**